playwright/Customizations/ComplianceWarnings.spec.ts

```diff
@@ -0,0 +1,327 @@
+  blueprintId: string,
+  blueprintName: string,
+  state: State,
+) => {
+  const distribution = 'rhel-9';
+  const image_requests: never[] = [];
+
+  if (state === 'A_only') {
+    return {
+      id: blueprintId,
+      name: blueprintName,
+      description: 'Compliance test blueprint',
+      distribution,
+      image_requests,
+      customizations: { packages: [] },
+      lint: {
+        errors: [
+          {
+            name: 'compliance',
+            description:
+              'Compliance: package aide required by policy is not present',
+          },
+        ],
+      },
+    };
+  }
+
+  if (state === 'A_and_B') {
+    return {
+      id: blueprintId,
+      name: blueprintName,
+      description: 'Compliance test blueprint',
+      distribution,
+      image_requests,
+      customizations: { packages: [] },
+      lint: {
+        errors: [
+          {
+            name: 'compliance',
+            description:
+              'Compliance: package audit required by policy is not present',
+          },
+          {
+            name: 'compliance',
+            description:
+              'Compliance: package aide required by policy is not present',
+          },
+        ],
+      },
+    };
+  }
+
+  if (state === 'audit_missing') {
+    return {
+      id: blueprintId,
+      name: blueprintName,
+      description: 'Compliance test blueprint',
+      distribution,
+      image_requests,
+      customizations: { packages: [] },
+      lint: {
+        errors: [
+          {
+            name: 'compliance',
+            description:
+              'Compliance: package audit required by policy is not present',
+          },
+        ],
+      },
+    };
+  }
+
+  if (state === 'audit_present') {
+    return {
+      id: blueprintId,
+      name: blueprintName,
+      description: 'Compliance test blueprint',
+      distribution,
+      image_requests,
+      customizations: { packages: ['audit'] },
+      lint: { errors: [] },
+    };
+  }
+
+  return {
+    id: blueprintId,
+    name: blueprintName,
+    description: 'Compliance test blueprint',
+    distribution,
+    image_requests,
+    customizations: { packages: [] },
+    lint: {
+      errors: [
+        {
+          name: 'compliance',
+          description:
+            'Compliance: package audit required by policy is not present',
+        },
+      ],
+    },
+  };
+};
+
+const mkBlueprintComposes = (distribution = 'rhel-9') => ({
+  meta: { count: 1 },
+  links: { first: '', last: '' },
+  data: [
+    {
+      id: uuidv4(),
+      request: {
+        distribution,
+        image_requests: [],
+      },
+      created_at: new Date().toISOString(),
+      blueprint_id: uuidv4(),
+      blueprint_version: 1,
+    },
+  ],
+});
+
+test.describe('Compliance warnings ignore flow', () => {
+  // Skip entire suite unless explicitly enabled with ENABLE_COMPLIANCE_IGNORE_FIX=1
+  const enableSuite = process.env.ENABLE_COMPLIANCE_IGNORE_FIX === '1';
```

Comment:
I can't see the variable being set anywhere, are the tests skipping this suite every time? Also why do we need to gate this?